jabgui/src/main/java/org/jabref/gui/edit/automaticfiededitor/clearcontent/ClearContentViewModel.java

```diff
@@ -0,0 +1,59 @@
+package org.jabref.gui.edit.automaticfiededitor.clearcontent;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.jabref.gui.StateManager;
+import org.jabref.gui.edit.automaticfiededitor.LastAutomaticFieldEditorEdit;
+import org.jabref.gui.undo.NamedCompoundEdit;
+import org.jabref.gui.undo.UndoableFieldChange;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.FieldFactory;
+
+public class ClearContentViewModel {
+    public static final int TAB_INDEX = 2;
```

Comment:
TAB_INDEX constant is horrible, but is actually consistent with the current design of the dialog. Neccessary for the fancy infotext about affected files after applying the change. Should be refactored in the future, but for now, it can just stay.